front/app/components/FilterBoxes/utils.tsx

```diff
@@ -0,0 +1,8 @@
+export const scrollToTopIdeasList = () => {
+  const ideasListPanel = document.getElementById('ideas-list-scroll-anchor');
+  ideasListPanel?.scrollIntoView({
+    behavior: 'smooth',
+    block: 'start',
+    inline: 'start',
```

Comment:
I'm not seeing changes if I adjust this to different values, so I suspect this can be left out as well.